test/unit/reference_extractor_test.rb

```diff
@@ -50,6 +50,15 @@ def setup
       assert_equal "::Order::Extension", reference.constant.name
     end
 
+    test "doesn't break on unusal constants it can't detect" do
```

Comment:
I guess you can make this test description more specific now, something about namespacing by `self`?

Reply:
good catch! Thanks!